02/app.js

```diff
@@ -1 +1,9 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+const links = document.querySelectorAll('[data-url]')
+
+if(links){
```

Comment:
Spędzanie `links` nie ma sensu bo to zawsze jest prawda (jest zwracana pusta tablica przez `querySelectorAll`, gdy nie zostały znlezione elementy)